civet/settings.py

```diff
@@ -308,8 +310,8 @@ def show_debug_toolbar(request):
         "api_url": "https://api.github.com",
         "html_url": "https://github.com",
         "hostname": "github.com",
-        "secret_id": "<secret_id>",
-        "client_id": "<client_id>",
+        "secret_id": "e8e92f3dbc25a20c9ec118af816e9cb7da91c5e5",
+        "client_id": "411bb44e8e7672a3f78b",
```

Comment:
Can we remove this? And Squash the commits so it never enters. Unless it is harmless (since it looks to be local). But still, it should be set back to an example default.